uPortal-webapp/src/main/webapp/WEB-INF/jsp/Marketplace/portlet/entry.jsp

```diff
@@ -18,6 +18,7 @@
     under the License.
 
 --%>
+<%@page contentType="text/html;charset=UTF-8"%>
```

Comment:
Why was this part needed again?

Does it matter that the output of this JSP will be combined into a bigger page with other markup that (almost certainly) didn't include this declaration?

Reply:
AFAICT the `charset` is being set in the XSLT.
The JSP also needs to know the content type, otherwise it breaks the ☆ symbols into three different icons.